src/bm_sim/P4Objects.cpp

```diff
@@ -2650,6 +2660,12 @@ P4Objects::map_json_value(const std::string &type, const std::string &name,
     } else {
       cfg_parse_states_map[cfg_parse_state_id] = val;
     }
+  } else if (type == "register_array") {
+    if (cfg_register_arrays_map.find(name) != cfg_register_arrays_map.end()) {
+      std::cout << "map_json_value: duplicated register_array name: " << name << std::endl;
```

Comment:
Consider throwing an error or other ways here if this will cause fatal errors in the following processing.